### PR TITLE
Remove status endpoints from getPlayerInfo

### PR DESCRIPTION
**Pull request by BillyH-0167**

### CODE REVIEW

**Comment by BillyH-0167** [comment]
Looks like someone else has already done a PR on 535 (https://github.com/noblox/noblox.js/pull/535)

File: typings/index.d.ts [hunk 1]
```diff
@@ -1191,8 +1191,7 @@ declare module "noblox.js" {
 
     interface PlayerInfo {
         username: string;
-        displayName: string;
-        status?: string;
+        displayName: string
```
Comment (suufi) [inline typings/index.d.ts:1194]: missing ;

**Review by suufi — CHANGES_REQUESTED** [review]

**Comment by alanbixby** [comment]
image

**Comment by BillyH-0167** [comment]
Fixed syntax issue. Should be all good to publish now.

**Comment by alanbixby** [comment]
Duplicate of #535 ; however, thank you for submitting.

**Pull request closed without merging**